test/compilable/line.d

```diff
@@ -6,19 +6,25 @@ int #line 10
 x;
 
 static assert(__LINE__ == 12);
-version(Windows)
+version(Windows) {
     static assert(__FILE__ == "compilable\\line.d");
-else
+    static assert(__FILE_FULL_PATH__[1..3] == ":\\");
+} else {
     static assert(__FILE__ == "compilable/line.d");
+    static assert(__FILE_FULL_PATH__[0] == '/');
+}
+static assert(__FILE_FULL_PATH__[$-__FILE__.length..$] == __FILE__);
 
 #line 100 "newfile.d"
 
 static assert(__LINE__ == 101);
 static assert(__FILE__ == "newfile.d");
+//static assert(__FILE_FULL_PATH__[$-__FILE__.length..$] == __FILE__);
```

Comment:
It looks like in this case `__FILE_FULL_PATH__` contains the absolute path to "line.d" as opposed to "newfile.d". I'm not sure exactly how this test is working...so I'm not sure what `__FILE_FULL_PATH__` should really do in this case.  Maybe it shouldn't match what `__FILE__` expands to?